### ISSUE TITLE
Cut/Copy/paste between windows

### ISSUE THREAD

**Issue by cseickel** [nvim-neo-tree/neo-tree.nvim#168]
Right now the cut/copy clipboard only works within a window. It might be nice to have a clipboard that is shared within a tab so that you can create two neo-tree windows in splits and be able to paste those items into a second tree.

**Comment by miversen33** [comment]
Yes. In fact, this could be a shared clipboard with neo-tree and any providers within it (netman would love this <3 )
